src/sweetalert2.js

```diff
@@ -1176,6 +1176,52 @@ sweetAlert.init = function() {
     sweetAlert.resetValidationError();
   };
 
+  /*
+   *  @summary IOS input focus fix
+   *  @var {iosfix}, {head} 
+   *  @description creating <style> tag with ios on focus fix
+   *  @author yeomann
+  */
+  var iosfix;
+  iosfix = document.createElement('style');
+  iosfix.type = "text/css";
+  iosfix.id = "iosFix"
+  iosfix.innerHTML = "html,body {height: 100%; position: relative; }";
```

Comment:
from my side I have now idea why this code should work :) Can you please attach the gif of swal2 working correctly on your iphone?